src/assets/getTextData.js

```diff
@@ -173,7 +173,7 @@ const getClientUrl = () => {
 const sendDataToClient = () => {
   const { selectionData, linkData } = getTextData(document.body);
   const iframeHeight = getPageHeight();
-  window.parent.postMessage({ selectionData, linkData, iframeHeight }, getClientUrl());
+  window.parent.postMessage({ type: 'selectionData', selectionData, linkData, iframeHeight }, getClientUrl());
```

Comment:
Can we change 'selectionData' to 'websiteData'?

Reply:
Updated